Core/RestaurantReservation.Application/Contracts/IRepository/ICustomerRepository.cs

```diff
@@ -0,0 +1,9 @@
+﻿using RestaurantReservation.Application.DTOs;
```

Comment:
In the clean archeticture we place the contracts/interfaces in the Domain layer.